server-landscape-corrected.yaml

```diff
@@ -0,0 +1,145 @@
+heat_template_version: 2013-05-23
```

Comment:
`server.yaml` has template version `2015-10-15`. Do you need the older one here?